### DISCUSSION TITLE
Delete device group

### DISCUSSION THREAD

**Discussion by mg1067** [Ylianst/MeshCentral#5602] [Q&A] [answered]
I have a three empty device groups. Now I like to delete them.. But there is no delete button.
How can I do this. 

I use Version 1.1.16

**Comment by si458** [comment]
Click the group name, then click delete group down the bottom right hand side
Or 
Click of the my account tab (second one down) then click on the group you want, then click delete group

**Comment by mg1067** [comment]
There  is no button. Thats the reason I ask.. Mhhmmm 

![image](https://github.com/Ylianst/MeshCentral/assets/88426250/69bec684-9ff3-446a-ae1d-5e5584345d55)

**Reply by si458** [reply]
I think it's because the is a user still assigned to the group but also is your user the admin user or a normal user

**Reply by mg1067** [reply]
My user is admin.
And all three groups are assigned to my user.

**Reply by si458** [reply]
Weird? Just trying to translate ui,
defo says Michael full rights
I'm guessing you are logged in as Michael? Stupid question sorry
Is the any other groups setup that maybe using a device that was in this group as a relay?
Have you tried simply restarting meshcentral? Stupid question sorry

**Comment by mg1067** [comment]
yes I am logged in as michael (admin). And yes I restart meshcentral. And no .. there is no other group to use it as relay. 
Is realy weird.... Something is wrong...

**Comment by silversword411** [comment]
![2023-12-02_115844 - mesh group delete](https://github.com/Ylianst/MeshCentral/assets/13395348/6a24facb-650e-4a4e-b38c-4d3745e14804)

Bottom right of edit group when you find what's blocking you. You have more than one group? You need at least one

**Comment by mg1067** [comment, marked as answer]
I know. Normally I do. But not for me. But I have now found the error. It's my admin user. I have now created a new admin user. Delete group is back. 

Thanks for the help anyway
